.github/workflows/dapr-pubsub.yaml

```diff
@@ -22,11 +22,6 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
 
-    - name: Set up Go 1.20
```

Comment:
oh i see, the run below doesn't need Go, right? Good catch!

Reply:
easy to tell since 1.20 (without quotes) gets interpreted by javascript as 1.2 🙃